Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM alpine:3.2
+
+RUN echo 'http://dl-4.alpinelinux.org/alpine/edge/main' >> /etc/apk/repositories \
```

Comment:
Only need to do this if we wanna run this in go 1.5 - 1.4 should be fine and thats whats in the stable packages.